@karllessard : Was just following up on this and noticed that the tests were failing.
Could you take a look and update the PR?

For example, the following is the log for `//tensorflow/java:GradientsTest`

```
tensorflow/java/src/test/java/org/tensorflow/op/core/GradientsTest.java:56: error: reference to feed is ambiguous
                  sess.runner().feed(x, c).fetch(grads.dy(0)).fetch(grads.dy(1)).run())) {
                               ^
  both method feed(Output<?>,Tensor<?>) in Session.Runner and method <T>feed(Operand<T>,Tensor<T>) in Session.Runner match
  where T is a type-variable:
    T extends Object declared in method <T>feed(Operand<T>,Tensor<T>)
tensorflow/java/src/test/java/org/tensorflow/op/core/GradientsTest.java:82: error: reference to feed is ambiguous
              new TestUtil.AutoCloseableList<>(sess.runner().feed(x, c).fetch(grads.dy(0)).run())) {
                                                            ^
  both method feed(Output<?>,Tensor<?>) in Session.Runner and method <T>feed(Operand<T>,Tensor<T>) in Session.Runner match
  where T is a type-variable:
    T extends Object declared in method <T>feed(Operand<T>,Tensor<T>)
tensorflow/java/src/test/java/org/tensorflow/op/core/GradientsTest.java:109: error: reference to feed is ambiguous
                  sess.runner().feed(x, c).fetch(grads1.dy(0)).run())) {
                               ^
  both method feed(Output<?>,Tensor<?>) in Session.Runner and method <T>feed(Operand<T>,Tensor<T>) in Session.Runner match
  where T is a type-variable:
    T extends Object declared in method <T>feed(Operand<T>,Tensor<T>)
```

--- a/tensorflow/java/src/main/java/org/tensorflow/Session.java
+++ b/tensorflow/java/src/main/java/org/tensorflow/Session.java
@@ -149,19 +149,9 @@ public final class Session implements AutoCloseable {
 
     /**
      * Use {@code t} instead of the Tensor referred to by executing the operation referred to by
-     * {@code output}.
-     */
-    public Runner feed(Output<?> o, Tensor<?> t) {
-      inputs.add(o);
-      inputTensors.add(t);
-      return this;
-    }
-
-    /**
-     * Use {@code t} instead of the Tensor referred to by executing the operation referred to by
      * {@code operand}.
      */
-    public <T> Runner feed(Operand<T> operand, Tensor<T> t) {
+    public Runner feed(Operand<?> operand, Tensor<?> t) {
       inputs.add(operand.asOutput());
       inputTensors.add(t);
       return this;

